doc/build_apps/js_app_bundle.rst

```diff
@@ -134,25 +129,35 @@ It validates the request body and returns the result of a mathematical operation
 Deployment
 ----------
 
-An app bundle can be wrapped into a governance proposal with the Python client for deployment:
+To deploy a new JS application, it must be embedded into a JSON governance proposal containing all of the new code and endpoint metadata. There is a tool in the CCF pip package to generate this object named ``bundle_js_app``.
```

Comment:
I think it's a "sample tool" but not necessarily how it must be done. The format of the proposal is standard and subject to release policy, the tool isn't and merely illustrates a possible usage.